browser_history/cli.py

```diff
@@ -16,44 +18,78 @@ def make_parser():
 
     :rtype: :py:class:`argparse.ArgumentParser`
     """
-    parser_ = argparse.ArgumentParser(description='''
+    parser_ = argparse.ArgumentParser(
+        description="""
                                             A tool to retrieve history from
-                                            (almost) any browser on (almost) any platform''',
-                                      epilog='''
+                                            (almost) any browser on (almost) any platform""",
+        epilog="""
                                             Checkout the GitHub repo https://github.com/pesos/browser-history
-                                            if you have any issues or want to help contribute''')
-
-    parser_.add_argument('-b', '--browser',
-                         default='all',
-                         help=f'''
-                                browser to retrieve history from. Should be one of all, {AVAILABLE_BROWSERS}.
-                                Default is all (gets history from all browsers).''')
-
-    parser_.add_argument('-f', '--format',
-                         default="csv",
-                         help=f'''
+                                            if you have any issues or want to help contribute""",
+    )
+
+    parser_.add_argument(
+        "-t",
+        "--type",
+        default="history",
+        help=f"""
+                                argument to decide whether to retrieve history or bookmarks.
+                                Should be one of all, {AVAILABLE_TYPES}.
+                                Default is history
+                                .""",
+    )
+    parser_.add_argument(
+        "-b",
+        "--browser",
+        default="all",
+        help=f"""
+                                browser to retrieve history or bookmarks from. Should be one of all, {AVAILABLE_BROWSERS}.
+                                Default is all (gets history or bookmarks from all browsers).""",
```

Comment:
Looks a little weird, remove some tabs/spaces here.

Reply:
black did that automatically for some reason

Reply:
`black` doesn't change indentation inside strings. This indent looked ok initially, but after black fixed the other indentation, it became weird